src/index.tsx

```diff
@@ -1,15 +1,40 @@
 import './wdyr'
 import 'lib/polyfills'
 
+import * as Sentry from '@sentry/react'
 import { App } from 'App'
 import { AppProviders } from 'AppProviders'
+import { getConfig } from 'config'
 import React from 'react'
 import { createRoot } from 'react-dom/client'
 import { renderConsoleArt } from 'lib/consoleArt'
 import { reportWebVitals } from 'lib/reportWebVitals'
 
 import * as serviceWorkerRegistration from './serviceWorkerRegistration'
 
+Sentry.init({
+  ...(window.location.hostname === 'localhost'
+    ? {}
+    : { dsn: getConfig().REACT_APP_SENTRY_DSN_URL }),
+  attachStacktrace: true,
+  denyUrls: ['alchemy.com'],
+  integrations: [
+    // Sentry.browserTracingIntegration(),
```

Comment:
The name is a bit misleading here - this is for perf. monitoring, we could enable this later if we want to https://docs.sentry.io/platforms/javascript/configuration/integrations/browsertracing/